src/contract.js

```diff
@@ -78,27 +82,36 @@ class Contract {
     }
 
     let dataHex = '';
-    dataHex = dataHex.concat(Utils.getFunctionHash(methodObj));
+    dataHex = dataHex.concat(Encoder.getFunctionHash(methodObj));
 
     let hex;
     _.each(methodObj.inputs, (item, index) => {
       switch (item.type) {
-        case 'address':
-          hex = Utils.addressToHex(args[index]);
+        case 'address': {
+          hex = Encoder.addressToHex(args[index]);
           dataHex = dataHex.concat(hex);
           break;
-        case 'bytes32[10]':
-          hex = Utils.stringArrayToHex(args[index], 10);
-          dataHex = dataHex.concat(hex);
+        } 
+        case 'bytes32[10]': {
```

Comment:
it should be able to handle arbitrary length of byte32 array. Leave it as it is for now, should add `//TODO` comment

Reply:
Plan to make it more variable later. Just getting it working for now.